.github/workflows/galaxy.yml

```diff
@@ -8,37 +8,7 @@ on:
 
 jobs:
   deploy:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v2
+    uses: T-Systems-MMS/.github/.github/workflows/ansible-galaxy-publish.yml@main
```

Comment:
I think this URL isn't correct anymore, the repo seemingly moved to the telekom-mms organisation in the meantime.